testapi.pm

```diff
@@ -789,15 +789,18 @@ sub is_serial_terminal {
 
 =head2 wait_serial
 
-  wait_serial($regex or ARRAYREF of $regexes [[, $timeout], $expect_not_found]);
+  wait_serial($regex or ARRAYREF of $regexes [[[, $timeout], $expect_not_found],
```

Comment:
Not sure about our convention but `[[[` looks wrong.

Reply:
Actually I think it should be the other way around. The only required argument is `$regexp` but one can only specify additional arguments if $expect_not_found and $timeout are both specified as they are `shift`-ed in first. A different thing on top: `no_regex` is no parameter of `wait_serial` but rather all additional arguments are forwarded. So I guess it should read

```suggestion
  wait_serial($regex or ARRAYREF of $regexes [, $timeout [, $expect_not_found [, @args ]]]);
```

similar to what we have for `select_console($console [, @args ])`

However we can still keep the explanation that `no_regex` as one of these parameters can be specified and what it does.